pydra/engine/task.py

```diff
@@ -485,6 +475,8 @@ def __init__(self, input_spec: ty.Optional[ShellSpec]=None,
     def command_args(self):
         args = []
         for f in dc.fields(self.inputs):
+            if f.name not in ['executable', 'args']:
+                continue
```

Comment:
we will have to end up including those metadata pieces in pydra, but i wanted to get the simplest options worked out first without formal input_specs.

i did mention to @mgxd that some of the initial enhancements we may want may be things like:

`sc.inputs.args = ['-f $fraction', '--nvols 2', '-o $out_file']`

and this would add `fraction` as an input, and `out_file` as both an input and an output.

but we would also be able to specify outputs using metadata on inputs and outputs. this will give us a chance to clean up autogeneration and globbing in the spec itself, with additional callables added as necessary when generation and globbing are not feasible. Examples for features we may want to support are:

```
'{input_arg1}_out.nii.gz' or '{input_xor_arg2}_full.nii.gz' 
'*.corr.out'
regexp 
mycollectorfunc()
```
but these output capture functionality would be added perhaps even for function interfaces, so in basetask.